Modules/Office365DSC/DSCResources/MSFT_SPOPropertyBag/readme.md

```diff
@@ -0,0 +1,5 @@
+# SPOPropertyBag
+
+## Description
+
+Configures a value in a site's property bag.
```

Comment:
Here you specify site property bag, but the parameter name is WebUrl. This doesn't sound consistent.

Reply:
Fixed by simply renaming Url